### ISSUE TITLE
What's New: Accessibility Review

### ISSUE THREAD

**Issue by rachelmcr** [woocommerce/woocommerce-ios#5274]
Review the following with the What's New component:

- [x] Dark mode
- [x] Dynamic type (large fonts)
- [x] VoiceOver / Spoken content
- [x] Landscape
- [x] iPad
- [x] Localization / RTL

**Comment by rachelmcr** [comment]
Dark mode is fine except for icons. The API that serves the announcements (including icons) doesn't account for light/dark mode — it only returns a single icon image. That should be fine for most icons; I mainly noticed because the test announcement uses black icons:

Light mode|Dark mode
-|-
![Simulator Screen Shot - iPhone 13 Pro - 2021-10-25 at 15 04 15](https://user-images.githubusercontent.com/8658164/138719128-2f2dd338-2931-44dd-8499-a7ed8150d990.png)|![Simulator Screen Shot - iPhone 13 Pro - 2021-10-25 at 15 05 02](https://user-images.githubusercontent.com/8658164/138719135-406dea97-1e41-40de-901a-eedf22895cbd.png)

There is some discussion about serving icons locally from the client, or adjusting the API to provide multiple icons, which we can consider for a V2 release (internal ref: p91TBi-5Wo-p2#comment-6220)

**Comment by rachelmcr** [comment]
VoiceOver and Spoken Content mostly works well. One issue I noticed:

- [ ] Icon should be hidden from VoiceOver since it's purely decorative
